### ISSUE TITLE
Unable to bind Own API Key

### ISSUE THREAD

**Issue by Gilbert2824** [zhangrenyang/chatgpt#27]
I have try to bind the own API key but its not working its giving the error. Error is " Failure, you've bound your own API key: TcsxcAknTcb*********************Ms956pJ60QsNufjS, its status is invalid and cannot be used. "

**Comment by quocbao238** [comment]
@Gilbert2824  Please help me fix this

**Comment by jkLennon** [comment]
I had the same problem

**Comment by kartikm4023** [comment]
Im facing the same problem - this issue is intermittent, not sure why it keeps happening.

**Comment by Julian-Won** [comment]
i have same problem

**Comment by Julian-Won** [comment]
when bind the API,the information that shows lost the api-header sk-****

**Comment by zhangrenyang** [comment]
> I have try to bind the own API key but its not working its giving the error. Error is " Failure, you've bound your own API key: TcsxcAknTcb*********************Ms956pJ60QsNufjS, its status is invalid and cannot be used. "

This is usually due to the unavailability of the API key, which is usually due to insufficient or expired account balance. Please log in to the official website to check the account status

1： Log in to the OpenAI website
To view the balance of the chatgpt API, you first need to log in to the official website of OpenAI.
access [https://www.openai.com](https://www.openai.com)  Find and click on the "Login" button in the upper right corner.
Enter your OpenAI account and password to log in.
If you do not already have an OpenAI account, you need to register first.
2： Enter the API Dashboard
After successful login, you will enter the OpenAI user interface.
Find the 'APIs' option in the navigation menu and click.
This will take you to the API Dashboard, where you can manage and view your API related information.
3： View API Key
In the API Dashboard, you can see the API key you have created.
Find the API key where you want to view the balance and click the "View" button next to it.
In the pop-up window, you will see detailed information about your API key, including the balance.
4： View API balance
In the detailed information of the API key, you can find your API balance.
The balance is usually displayed in the form of available request quantity or available monetary amount.
You can use the balance information to understand how many resources you have left to use.
5： Recharge and manage balance
If your API balance is insufficient, you can increase the balance by recharging.
In the API Dashboard, locate the "Billing" option and click.
This will take you to the billing page of OpenAI, where you can manage your balance and payment method.
According to your needs, you can choose to recharge or set automatic recharge options to maintain your API balance.

**Comment by zhangrenyang** [comment]
> @Gilbert2824 Please help me fix this


This is usually due to the unavailability of the API key, which is usually due to insufficient or expired account balance. Please log in to the official website to check the account status

1： Log in to the OpenAI website
To view the balance of the chatgpt API, you first need to log in to the official website of OpenAI.
access [https://www.openai.com](https://www.openai.com)  Find and click on the "Login" button in the upper right corner.
Enter your OpenAI account and password to log in.
If you do not already have an OpenAI account, you need to register first.
2： Enter the API Dashboard
After successful login, you will enter the OpenAI user interface.
Find the 'APIs' option in the navigation menu and click.
This will take you to the API Dashboard, where you can manage and view your API related information.
3： View API Key
In the API Dashboard, you can see the API key you have created.
Find the API key where you want to view the balance and click the "View" button next to it.
In the pop-up window, you will see detailed information about your API key, including the balance.
4： View API balance
In the detailed information of the API key, you can find your API balance.
The balance is usually displayed in the form of available request quantity or available monetary amount.
You can use the balance information to understand how many resources you have left to use.
5： Recharge and manage balance
If your API balance is insufficient, you can increase the balance by recharging.
In the API Dashboard, locate the "Billing" option and click.
This will take you to the billing page of OpenAI, where you can manage your balance and payment method.
According to your needs, you can choose to recharge or set automatic recharge options to maintain your API balance.

**Comment by zhangrenyang** [comment]
> I had the same problem

This is usually due to the unavailability of the API key, which is usually due to insufficient or expired account balance. Please log in to the official website to check the account status

1： Log in to the OpenAI website
To view the balance of the chatgpt API, you first need to log in to the official website of OpenAI.
access [https://www.openai.com](https://www.openai.com)  Find and click on the "Login" button in the upper right corner.
Enter your OpenAI account and password to log in.
If you do not already have an OpenAI account, you need to register first.
2： Enter the API Dashboard
After successful login, you will enter the OpenAI user interface.
Find the 'APIs' option in the navigation menu and click.
This will take you to the API Dashboard, where you can manage and view your API related information.
3： View API Key
In the API Dashboard, you can see the API key you have created.
Find the API key where you want to view the balance and click the "View" button next to it.
In the pop-up window, you will see detailed information about your API key, including the balance.
4： View API balance
In the detailed information of the API key, you can find your API balance.
The balance is usually displayed in the form of available request quantity or available monetary amount.
You can use the balance information to understand how many resources you have left to use.
5： Recharge and manage balance
If your API balance is insufficient, you can increase the balance by recharging.
In the API Dashboard, locate the "Billing" option and click.
This will take you to the billing page of OpenAI, where you can manage your balance and payment method.
According to your needs, you can choose to recharge or set automatic recharge options to maintain your API balance.

**Comment by zhangrenyang** [comment]
> Im facing the same problem - this issue is intermittent, not sure why it keeps happening.


This is usually due to the unavailability of the API key, which is usually due to insufficient or expired account balance. Please log in to the official website to check the account status

1： Log in to the OpenAI website
To view the balance of the chatgpt API, you first need to log in to the official website of OpenAI.
access [https://www.openai.com](https://www.openai.com)  Find and click on the "Login" button in the upper right corner.
Enter your OpenAI account and password to log in.
If you do not already have an OpenAI account, you need to register first.
2： Enter the API Dashboard
After successful login, you will enter the OpenAI user interface.
Find the 'APIs' option in the navigation menu and click.
This will take you to the API Dashboard, where you can manage and view your API related information.
3： View API Key
In the API Dashboard, you can see the API key you have created.
Find the API key where you want to view the balance and click the "View" button next to it.
In the pop-up window, you will see detailed information about your API key, including the balance.
4： View API balance
In the detailed information of the API key, you can find your API balance.
The balance is usually displayed in the form of available request quantity or available monetary amount.
You can use the balance information to understand how many resources you have left to use.
5： Recharge and manage balance
If your API balance is insufficient, you can increase the balance by recharging.
In the API Dashboard, locate the "Billing" option and click.
This will take you to the billing page of OpenAI, where you can manage your balance and payment method.
According to your needs, you can choose to recharge or set automatic recharge options to maintain your API balance.

**Comment by wmosquini2** [comment]
Has anyone managed to configure the API?
I can use the free API elsewhere, but I can't use it in the Visual Code extension.

**Comment by lmarcelocc** [comment]
> Has anyone managed to configure the API? I can use the free API elsewhere, but I can't use it in the Visual Code extension.

nop, is not working here as wel

**Comment by dukartbr** [comment]
Having the same issue

**Comment by dukartbr** [comment]
Was having the same issue but turned out to be user error. There is a difference between ChatGPT's premium, and Open AI API usage. I had to do some digging to find out the difference, they don't make the differences very clear in the ChatGPT upgrade payment UI. I was able to upgrade on my [Open AI dashboard](https://platform.openai.com/account/billing/overview) and the API key worked after that. Hope this helps anyone who's tearing their hair out.
